ci-operator/step-registry/ipi/conf/gcp/gpu-computing-node/ipi-conf-gcp-gpu-computing-node-ref.yaml

```diff
@@ -0,0 +1,26 @@
+ref:
+  as: ipi-conf-gcp-gpu-computing-node
```

Comment:
is there anything that is specific to GCP in this step? If not, perhaps we can remove it from the name of the step